backend/.env-example

```diff
@@ -63,10 +69,14 @@ TELEGRAM_BOT_GUIDE_URL="https://go.gov.sg/postman-recipient-guide"
 # FILE_STORAGE_BUCKET_NAME="localstack-upload"
 # AWS_LOG_GROUP_NAME="postmangovsg-beanstalk-localstack"
 
+# DOMAIN_WHITELIST=".gov.sg"
+# COOKIE_DOMAIN="localhost"
+
 # FRONTEND_URL="http://localhost:3000"
 # SENTRY_DSN="https://123.ingest.sentry.io/456"
 
-# set of twilio credentials used for testing locally only
+# Note 1: set of twilio credentials used for testing locally only
+# Note 2: demo Twilio credentials are fetched from AWS Secrets Manager; NOT from env vars
```

Comment:
can we just remove these bunch altogether, they're not helpful at all in anyway

Reply:
I suspect they were used to make test API calls to Twilio in other parts of the codebase 🤔

I'll remove it here but keep it in the 1Password Vault in case it becomes helpful in the future.